tests/Extension/EscaperTest.php

```diff
@@ -182,6 +182,15 @@ public function testJavascriptEscapingConvertsSpecialChars()
         }
     }
 
+    public function testJavascriptEscapingConvertsSpecialCharsWithInternalEncoding()
+    {
+        $twig = new Environment($this->createMock(LoaderInterface::class));
+        mb_internal_encoding('ISO-8859-1');
```

Comment:
this should be reset to the previous value in a `finally` block, to avoid impacting other tests

Reply:
Done